porch/controllers/packagevariantsets/pkg/controllers/packagevariantset/packagevariantset_controller.go

```diff
@@ -98,7 +111,20 @@ func (r *PackageVariantSetReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	downstreams, err := r.unrollDownstreamTargets(ctx, pvs,
 		upstream.Package)
 	if err != nil {
-		return ctrl.Result{}, err
+		if strings.Contains(err.Error(), "unable to list specified objects in cluster") {
+			conditions = append(conditions, metav1.Condition{
+				Type:               "ObjectsListed",
```

Comment:
Do we need a separate conditions for this? If the user has specified an unknown resource type it seems like that could be handled as a validation error?

Reply:
Changed it to be part of the Valid condition.